public/app/plugins/datasource/influxdb/components/RawInfluxQLEditor.test.tsx

```diff
@@ -57,7 +57,7 @@ describe('RawInfluxQLEditor', () => {
     const formatSelect = screen.getByLabelText('Format as');
     expect(formatSelect).toBeInTheDocument();
 
-    await select(formatSelect, 'Time series');
+    await select(formatSelect, 'Time series', { container: document.body });
```

Comment:
this seems like a big pain to always have to specify in tests

Reply:
Maybe add a `select` function to grafana-ui that basically just does this by default?